lib/js-api/tests/externals.rs

```diff
@@ -0,0 +1,418 @@
+use wasm_bindgen_test::*;
+// use anyhow::Result;
+use wasmer_js::*;
+
+#[wasm_bindgen_test]
+fn global_new() {
+    let store = Store::default();
+    let global = Global::new(&store, Value::I32(10));
+    assert_eq!(
+        *global.ty(),
+        GlobalType {
+            ty: Type::I32,
+            mutability: Mutability::Const
+        }
+    );
+
+    let global_mut = Global::new_mut(&store, Value::I32(10));
+    assert_eq!(
+        *global_mut.ty(),
+        GlobalType {
+            ty: Type::I32,
+            mutability: Mutability::Var
+        }
+    );
+}
+
+#[wasm_bindgen_test]
+fn global_get() {
+    let store = Store::default();
+    let global_i32 = Global::new(&store, Value::I32(10));
+    assert_eq!(global_i32.get(), Value::I32(10));
+    // let global_i64 = Global::new(&store, Value::I64(20));
+    // assert_eq!(global_i64.get(), Value::I64(20));
+    let global_f32 = Global::new(&store, Value::F32(10.0));
+    assert_eq!(global_f32.get(), Value::F32(10.0));
+    // let global_f64 = Global::new(&store, Value::F64(20.0));
+    // assert_eq!(global_f64.get(), Value::F64(20.0));
```

Comment:
Why is it commented?

Reply:
I added a comment to the code clarifying:
```
    // 64-bit values are not yet fully supported in some versions of Node
    // Commenting this tests for now:
```